android/app/src/main/java/org/fossasia/openevent/core/feed/facebook/CommentsListAdapter.java

```diff
@@ -55,28 +55,30 @@ public RecyclerViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
     @Override
     public void onBindViewHolder(RecyclerViewHolder holder, int position) {
 
-        final CommentItem commentItem = commentItems.get(position);
+        final CommentItem comment = commentItems.get(position);
 
-        if (commentItem.getCreatedTime() != null) {
-            String createdTime = commentItem.getCreatedTime();
+        if (comment.getCreatedTime() != null) {
+            String createdTime = comment.getCreatedTime();
             try {
                 holder.commentTime.setText(DateConverter.getRelativeTimeFromTimestamp(createdTime));
             } catch (DateTimeParseException e) {
                 Timber.e(e);
             }
         }
 
-        if (!TextUtils.isEmpty(commentItem.getMessage())) {
-            holder.comment.setText(commentItem.getMessage());
+        if (!TextUtils.isEmpty(comment.getMessage())) {
+            holder.comment.setText(comment.getMessage());
             holder.comment.setVisibility(View.VISIBLE);
         } else {
             // status is empty, remove from view
             holder.comment.setVisibility(View.GONE);
         }
 
-        if (!TextUtils.isEmpty(commentItem.getFrom().getName())) {
-            holder.commenter.setText(commentItem.getFrom().getName());
-            holder.commenter.setVisibility(View.VISIBLE);
+        if (comment.getFrom() != null) {
```

Comment:
Still here. I meant

```Java
From from = comment.getFrom();
if (from != null) {
    String name = from.getName();
    if(!TextUtils.isEmpty(name)) {
           holder.commenter.setText(name);
           holder.commenter.setVisibility(View.VISIBLE);
       }
}
```

Also, why not combine the two `if`s by AND?

Also, rever other changes

Reply:
Ohh! I will do thatv :+1:

Reply:
PR updated